rareapi/models/reaction.py

```diff
@@ -0,0 +1,4 @@
+from django.db import models
+
+class Reaction(models.Model):
+    emoji = models.CharField(max_length=50)
```

Comment:
Could you make the max_length longer to account for long ass URL's that might be put there? I'm not sure how we will use that, but it may be an url, may be a character set code in the future.